src/libs/DebugUtils.ts

```diff
@@ -271,7 +294,7 @@ function validateBoolean(value: string) {
  * Validates if a string is a valid representation of a date.
  */
 function validateDate(value: string) {
```

Comment:
Did you update the unit tests to cover the changes?

Reply:
I did not but I'll confirm if it's needed, thanks! 😄